### PR TITLE
🎨 Improve the service startup progress display

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?
Show pulling size Prefixes and persist decimals - (Optional as default is collapsed)

Screenshot 2024-04-29 at 16 40 13

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by pcrespov — COMMENTED** [review]
I have a small observation regarding formatting. Shouldn't you have a small space after the ellipses? For instance, instead of 
```
- Doing something...
```
I would propose
```
- Doing something ...
```

And second. 
Why a size is indicative of progress for "Installing a software"? I can understand that metric in things like "dowloading files" but not in installing software. I know it refers to the size of the image pulled but not every external user knows about this detail. Wouldnt' it make more sense to add a percentage and eventually a tooltip with extra info e.g. software X with v1.2 installed with size ...

**Review by pcrespov — COMMENTED** [review]
TIP: please change the PRs from DRAFT to READY to review before assigning them

File: services/static-webserver/client/source/class/osparc/utils/Utils.js [hunk 1]
```diff
@@ -552,7 +552,7 @@ qx.Class.define("osparc.utils.Utils", {
       return L > 0.35 ? "#FFF" : "#000";
     },
 
-    bytesToSize: function(bytes, decimals = 2, isPrefixVisible = true) {
+    bytesToSize: function(bytes, decimals = 2, isDecimalColapesed = true) {
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/utils/Utils.js:555]: MINOR: typo in `isDecimalColapesed` should be `isDecimalColapsed`

**Review by sanderegg — APPROVED** [review]
thanks!

**Comment by jsaq007** [comment]
> TIP: please change the PRs from DRAFT to READY to review before assigning them

Sorry it was ready I guess GitHub remembers previous use

**Review by odeimaiz — APPROVED** [review]

**Comment by jsaq007** [comment]
> And second. Why a size is indicative of progress for "Installing a software"? I can understand that metric in things like "dowloading files" but not in installing software. I know it refers to the size of the image pulled but not every external user knows about this detail. Wouldnt' it make more sense to add a percentage and eventually a tooltip with extra info e.g. software X with v1.2 installed with size ...

@pcrespov I a personally think this is Okay for the moment but would be up for discussing a better way forward. We now have possibilities if required to break it down further and really show the steps we are going through but there are other factors to consider therefore I decided to leave it a minimum for the moment. If the POs what me to go further in I will do that and perhaps we can discuss some of the finer details?

**Pull request merged**